lib/libsrtp.ex

```diff
@@ -0,0 +1,108 @@
+defmodule LibSRTP do
+  @moduledoc """
+  [LibSRTP](https://github.com/cisco/libsrtp) bindings for Elixir.
+
+  The workflow goes as follows:
+  - create LibSRTP instance with `new/0`
+  - add streams with `add_stream/2`
+  - protect or unprotect packets with `protect/3`, `unprotect/3`, `protect_rtcp/3`, `unprotect_rtcp/3`
+  - remove streams with `remove_stream/2`
+  """
+  alias LibSRTP.{Native, Policy}
+
+  require Record
+
+  @opaque t :: {__MODULE__, reference}
```

Comment:
After understanding this it looks so cool but I would make naming more consistent. Here we use `reference` in `ref` macro we use `native` and in function clauses we use `state`.  Looking at function definitions in `spec.exs` they all get `state` so maybe call this `state` in all places?